MaterialDesignThemes.Wpf/PackIcon.cs

```diff
@@ -17,9 +17,11 @@ public class PackIcon : PackIconBase<PackIconKind>
         static PackIcon()
         {
             DefaultStyleKeyProperty.OverrideMetadata(typeof(PackIcon), new FrameworkPropertyMetadata(typeof(PackIcon)));
-        }     
+        }
 
         public PackIcon() : base(PackIconDataFactory.Create)
-        { }    
+        {
+            FlowDirection = FlowDirection.LeftToRight;
```

Comment:
@punker76 @ButchersBoy you are right. I should move it to Generic.xaml.
Do I have to revert it in here? or make another commit?